docs-ref-conceptual/azure-cli-vm-tutorial-5.md

```diff
@@ -22,13 +22,25 @@ az network nic show --ids $nicId
 
 This command shows all of the information for the network interface of the VM. This data includes DNS settings, IP information, security settings, and the MAC address. The following query shows how to obtain the public IP address and subnet object IDs.
 
+# [Bash](#tab/bash)
+
 ```azurecli-interactive
 az network nic show --ids $nicId \
-  --query '{IP:ipConfigurations[].publicIpAddress.id, Subnet:ipConfigurations[].subnet.id}' \
+  --query '{IP:ipConfigurations[].publicIPAddress.id, Subnet:ipConfigurations[].subnet.id}' \
```

Comment:
@v-abdriaz , This command returned no results b/c JMESPath is case sensitive.  `publicIpAddress` is not the same as `publicIPAddress`.  If I recall correctly, this spelling was changed after this article was originally written, but having an empty `$nicId` makes the following steps not work correctly.

In the following screen print, there was no error, but the command didn't work.  When you test a code block, please verify both **errors and results**.

![image](https://github.com/MicrosoftDocs/azure-docs-cli/assets/58762114/d7527564-b35b-4374-918a-a2a6a9ed02b6)

I have fixed this spelling in your branch so I can merge the PR.